.github/workflows/ci.yml

```diff
@@ -149,15 +131,16 @@ jobs:
           SCCACHE_CACHE_SIZE: 128M
           SCCACHE_IDLE_TIMEOUT: 0
         run: |
-          $version = "0.2.12"
+          $version = "v0.8.2"
           $platform =
-            @{ "macOS"   = "x86_64-apple-darwin"
-               "Linux"   = "x86_64-unknown-linux-musl"
-               "Windows" = "x86_64-pc-windows-msvc"
-             }.${{ runner.os }}
+            @{ "x86_64-apple-darwin"       = "x86_64-apple-darwin"
+               "aarch64-apple-darwin"      = "aarch64-apple-darwin"
+               "x86_64-unknown-linux-gnu"  = "x86_64-unknown-linux-musl"
+               "aarch64-unknown-linux-gnu" = "aarch64-unknown-linux-musl"
+               "x86_64-pc-windows-msvc"    = "x86_64-pc-windows-msvc"
+             }['${{ matrix.config.target }}']
           $basename = "sccache-$version-$platform"
-          $url = "https://github.com/mozilla/sccache/releases/download/" +
-                 "$version/$basename.tar.gz"
+          $url = "https://github.com/mozilla/sccache/releases/download/$version/$basename.tar.gz"
```

Comment:
hi, can you help me understand, why use pwsh scripts to manually download sccache, instead of using the [sccache-action](https://github.com/Mozilla-Actions/sccache-action) ?

Reply:
This all was written before that existed. Glad to know about it!

Reply:
I tested with sccache-action in my fork repo: https://github.com/linrongbin16/rusty_v8/pull/2

The ci job failed on windows: https://github.com/linrongbin16/rusty_v8/actions/runs/14051568959

The error says 'sccahce' cannot compile 'syn' lib, and I search such kind of error, found this issue: https://github.com/mozilla/sccache/issues/2196

So the conclusion is: sccache has a different behavior for `rustc_wrapper` and server/client. So maybe keep it this way.

Reply:
I submit PR #1748 to simplify the installtion of `sccache`.

Reply:
Thanks - happy to land that patch if you can get it to work

Reply:
I did finished the PR, would you take a look?